src/components/OSCALControlImplementationImplReq.test.js

```diff
@@ -40,10 +40,19 @@ export default function testOSCALControlImplementationImplReq(
 
   test(`${parentElementName} displays component parameters in control prose`, () => {
     renderer();
-    const result = getByTextIncludingChildern(
-      "Does something with control 1 / component 1 / parameter 1 value and control 1 / component 1 / parameter 2 value"
+    const nonplaceholder1 = getByTextIncludingChildern(/does something with/i);
+    const placeholderText1 = getByTextIncludingChildern(
+      /control 1 \/ component 1 \/ parameter 1 value/i
     );
-    expect(result).toBeVisible();
+    const nonplaceholder2 = getByTextIncludingChildern(/and/i);
+    const placeholderText2 = getByTextIncludingChildern(
+      /control 1 \/ component 1 \/ parameter 2 value/i
+    );
```

Comment:
I see you're handling these parameters with regex. For consistency purposes among our other tests, could these be handled as strings?

Reply:
We have to options to make the tests consistent which do you think is better?
1. Convert the Regexp to strings
2. Convert the Strings to Regexp

Reply:
Convert the regex to strings in the two methods handled in this PR. It would be a lot less work.

Reply:
`DefinitionControlImplementation` test was able to convert to a string 
`ControlImplementationImplReq` doesn't work with strings. Also the linter will complain with the `\` which escape the `/`
I wouddn't mind converting the strings to RegExps for consistency purposes.

Reply:
Interesting, I would just say to leave that single one as a regex for now. Don't worry about converting the other string parameters to Regex.